docs/how-tos/how-to-reset-state.mdx

```diff
@@ -34,7 +34,7 @@ Alternatively, you can store the object in another object, which make the reset
 const state = proxy({ obj: initialObj })
 
 const reset = () => {
-  state.obj = cloneDeep(initialObj)
+  state.obj = deepClone(initialObj)
```

Comment:
😄 

Can you also fix L22 `deepCLone`?

Reply:
lol - I can type, I swear. Pushed up a fix.